libtsuba/include/tsuba/FileView.h

```diff
@@ -110,6 +110,7 @@ class KATANA_EXPORT FileView : public arrow::io::RandomAccessFile {
   }
 
   uint64_t size() const { return file_size_; }
+  std::string filename() const { return filename_; }
```

Comment:
This kind of accessor would usually return `const std::string&`; that way it's up to the caller whether an actual copy is needed or not.